### DISCUSSION TITLE
React / Elm - front end apps in a retype document

### DISCUSSION THREAD

**Discussion by benkoshy** [retypeapp/retype#396] [Q&A] [answered]
Hi there

I would like to deploy a small front end javascript app into a retype markdown document.

```html

<--  demo.md -->
<div id="react_app"> </div>
```
```js
// initialize_react.js

            // JS to initialise react app
            // JS for the react app itself
```

### Why?

I want to demonstrate,what the [pagy library](https://benkoshy.github.io/pagy/quick-start/) can do with a real life demonstration.

Is this possible?

**Comment by geoffreymcgill** [comment, marked as answer]
Hi @benkoshy. Unfortunately, Retype was not designed to support this scenario.
